source/find_getmore_killcursors_commands.rst

```diff
@@ -198,7 +198,7 @@ closed on 5.0:
     {getMore: ..., batchSize:1} // Returns remaining items but leaves cursor open on 5.0+
     {...}          // Kills server-side cursor. Necessary on 5.0+
 
-.. _CRUD: https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#id16
+.. _CRUD: ./crud/crud.md#id16
```

Comment:
```suggestion
.. _CRUD: ./crud/crud.md#find
```
Since the TOC is gone we should point this at the header

Reply:
This should never have used `#id16`. Those are the URLs you end up with after clicking a heading within the document, but any links from external sources should always point directly to the content.